cli/cliui/select_test.go

```diff
@@ -101,6 +101,39 @@ func TestMultiSelect(t *testing.T) {
 		}()
 		require.Equal(t, items, <-msgChan)
 	})
+
+	t.Run("MultiSelectWithCustomInput", func(t *testing.T) {
```

Comment:
couldn't add tests to check the custom input flow, we had this check here and not sure if its safe to remove and add interactive tests, please do let me know if there's a better way to incorporate those tests

https://github.com/coder/coder/blob/63572d9f5370a860653adf9ff61256a7aed1724c/cli/cliui/select.go#L310

Reply:
I think historically we've manually tested this using `prompt-example`. I agree that it's not ideal :-(